lib/storage/src/content_manager/collection_meta_ops.rs

```diff
@@ -152,6 +156,13 @@ pub struct ChangeAliasesOperation {
 #[serde(rename_all = "snake_case")]
 pub struct DeleteCollectionOperation(pub String);
 
+#[derive(Debug, Deserialize, Serialize, JsonSchema, PartialEq, Eq, Hash)]
+pub enum ShardTransferOperations {
+    Start { to: PeerId },
+    Finish,
+    Abort,
```

Comment:
What do you think about adding an error or description regarding what caused the `Abort`?